packages/ui/src/components/Banner/banner.tsx

```diff
@@ -38,6 +42,40 @@ const getSize =  (width: number) => {
     }
 }
 
+const getTextSize = (width: number, bannerSize: BannerProps['size']) => {
```

Comment:
Is it possible to use existing breakpoints?

Reply:
Banner has its own breakpoints with their own logic. I used them